src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -310,15 +310,19 @@ private function getCountFromQuery(Builder $query)
         // create an "outer" query, the one that is responsible to do the count of the "crud query".
         $outerQuery = $crudQuery->newQuery();
 
+        // in this outer query we will select only one column to be counted.
+        $outerQuery = $outerQuery->select($this->model->getKeyName());
+
         // add the count query in the "outer" query.
-        $outerQuery = $outerQuery->selectRaw("count('".$this->model->getKeyName()."') as total_rows");
+        $outerQuery = $outerQuery->selectRaw("count(*) as total_rows");
```

Comment:
Hmm... any reason to use `selectRaw` here? Can't we just do a `count()` and avoid raw queries, so it'd also work on MongoDB?